### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by callihiggins**

### CODE REVIEW

File: CoopANDlastfm.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
     		</select>
 		</form>
 		<div id="resubmit">resubmit!</div>
+		<div id="save">save</div>
```
Comment (callihiggins) [inline CoopANDlastfm.html:39]: Save button

File: coopAndLastFM2.js [hunk 2]
```diff
@@ -61,11 +61,11 @@ var app = {
 					console.log(searchResults);
 
 					//you can call a p5 function from here!
-					//loadTheImage(searchResults);
```
Comment (callihiggins) [inline coopAndLastFM2.js:64]: Turn off the old html rendering of images

File: coopAndLastFM2.js [hunk 3]
```diff
@@ -61,11 +61,11 @@ var app = {
...
-					var img = $('<img />',
-					{src: searchResults})
-					.appendTo($('#images'));
-					app.searchCoopHew(data);
+					loadTheImage(searchResults);
+					// var img = $('<img />',
```
Comment (callihiggins) [inline coopAndLastFM2.js:65]: load the image when we get it

File: coopAndLastFM2.js [hunk 4]
```diff
@@ -61,11 +61,11 @@ var app = {
 					console.log(searchResults);
 
 					//you can call a p5 function from here!
-					//loadTheImage(searchResults);
-					var img = $('<img />',
-					{src: searchResults})
-					.appendTo($('#images'));
-					app.searchCoopHew(data);
```
Comment (callihiggins) [inline coopAndLastFM2.js:68]: Don't call the function recursively

File: coopAndLastFM2.js [hunk 5]
```diff
@@ -137,50 +137,32 @@ var app = {
 			$("#searchResults").resizable();
 		});
 		
+		$("#save").click(function() {
+
+			saveCanvas('test', 'jpg');
+
+		});
```
Comment (callihiggins) [inline coopAndLastFM2.js:145]: Save the canvas when the save button is clicked

File: coopAndLastFM2.js [hunk 6]
```diff
@@ -137,50 +137,32 @@ var app = {
...
 
 	},
 	
 
 };
 
 // global variable to hold onto the image
-<<<<<<< HEAD
-//var theImage;
-=======
```
Comment (callihiggins) [inline coopAndLastFM2.js:152]: These are all merge conflicts, some things you have to pay attention to when you collaborate on a repo. Don't worry too much about this right now

**Review by izzylee0 — APPROVED** [review]

**Pull request merged**